src/Math/babylon.math.ts

```diff
@@ -278,6 +278,21 @@
             return new Color3(r / 255.0, g / 255.0, b / 255.0);
         }
 
+        /**
+         * Creates a new Color3 with values clamped between min and max.
+         * @param color - original color
+         * @param min - minimum value
+         * @param max - maximum value
+         * @returns - Color3
+         */
+        public static Clamp(color: Color3, min: number = 0, max: number = 1): Color3 {

```

Comment:
I would have seen these two functions as non static ones and named them clampToRef perhaps?

Reply:
Sure I can do that.